app/os_js.go

```diff
@@ -24,6 +24,14 @@ import (
 
 type ViewEvent struct{}
 
+type contextStatus uint8
```

Comment:
FWIW, using `uint8` for enums is (my) bad idea. A plain `int` is clearer and just as performant in most cases.